Resources/Locale/en-US/chat/managers/chat-manager.ftl

```diff
@@ -13,8 +13,15 @@ chat-manager-whisper-headset-on-message = You can't whisper on the radio!
 chat-manager-server-wrap-message = SERVER: {"{0}"}
 chat-manager-sender-announcement-wrap-message = {$sender} Announcement:
                                                 {"{0}"}
-chat-manager-entity-say-wrap-message = {$entityName} says: "{"{0}"}"
-chat-manager-entity-whisper-wrap-message = {$entityName} whispers: "{"{0}"}"
+chat-manager-entity-death-message = {$entityName}'s body seizes up and falls limp, his eyes dead and lifeless...
```

Comment:
Use pronouns.

Reply:
specifically, `{POSS-ADJ($entityName)}` instead of "his"